www/class/config-generate/broker.class.php

```diff
@@ -547,6 +547,19 @@ private function rpnOperation($result, $item)
         return $result;
     }
 
+    /**
+     * Method retrieving the Centreon Platform UUID generated during web installation
+     *
+     * @return string
+     */
+    private function getCentreonPlatformUuid(): string
+    {
+        global $pearDB;
+        $result = $pearDB->query("SELECT `value` FROM informations WHERE `key` = 'uuid'");
+        $record = $result->fetch(\PDO::FETCH_ASSOC);
```

Comment:
return null to handle the case where the record is false and add pearDB as parameter to the function